blocks-config/uagb-controls/addInitialAttr.js

```diff
@@ -37,15 +37,15 @@ const addInitialAttr = ( ChildComponent ) => {
 		]; // Add all parent block name here who's getting issue in customize preview.
 
 		useEffect( () => {
-			if ( uagb_blocks_info.is_customize_preview && undefined === block_id && listOfParentBlock.includes( name ) ) {
+			if ( uagb_blocks_info.is_customize_preview && ( '0' === block_id || undefined === block_id ) && listOfParentBlock.includes( name ) ) {
```

Comment:
Will this work as intended?

```suggestion
			if ( uagb_blocks_info.is_customize_preview && empty( block_id ) && listOfParentBlock.includes( name ) ) {
```

Reply:
@bsf-zanev Its JS and empty() function not available in JS.